.github/workflows/t3000-demo-tests-impl.yaml

```diff
@@ -10,9 +10,9 @@ jobs:
       matrix:
         test-group: [
           { name: "t3k_falcon40b_tests", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 50, owner_id: U053W15B6JF}, #Djordje Ivanovic
-          { name: "t3k_llama3_70b_tests", arch: wormhole_b0, cmd: run_t3000_llama3_70b_tests, timeout: 30, owner_id: U03FJB5TM5Y}, #Colman Glagovich
```

Comment:
Haha not sure why this was moved around but whatever

Reply:
Just to fix the order of smaller llama3 to large llama3 :) 
I know it doesn't change the order of pickup by a machine, but makes more sense when looking at the tests in the pipeline.